src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-serverId.html

```diff
@@ -1,4 +1,4 @@
 <div>
-    Select a Bitbucket Server from the list. If you cannot see your server listed, check the System Configuration
-    (Manage Jenkins -> Configure System).
+    This is the Bitbucket instance that contains the project you want Jenkins to build from. If you cannot see your
+    instance check your credentials and try again.
```

Comment:
/nit "check your configuration" ? 

I'm thinking there could be other reasons that the instance might not be listed, so perhaps being more generic and asking to check config rather than _just_ credentials might be better.